openmetadata-docs/content/v1.5.x/deployment/security/ldap/kubernetes.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Ldap Authentication for Kubernetes
+slug: /deployment/security/ldap/kubernetes
+collate: false
+---
+
+# LDAP Authentication for Kubernetes
+
+This guide outlines how to configure LDAP authentication for Kubernetes deployments of OpenMetadata. It includes details on required configurations, optional settings, and best practices to ensure secure and efficient authentication.
+
+## Authentication Configuration
+
+Update the `openmetadata.yaml` file with the following settings to enable LDAP authentication:
+
+```yaml
+global:
+  authentication:
+    provider: ldap
+    enableSelfSignup: false
+    publicKeyUrls:
+      - "https://<your-domain>/api/v1/system/config/jwks" # Replace with your domain
+    authority: "https://<your-domain>" # Replace with your domain
+    ldapConfiguration:
+      host: "ldap.example.com"  # Replace with your LDAP server hostname
+      port: 636  # Use 636 for secure LDAP (LDAPS) or 389 for standard LDAP
+      dnAdminPrincipal: "cn=admin,dc=example,dc=com"  # LDAP admin user DN
+      dnAdminPassword: "your-secret-password"  # Password for the admin user
+      userBaseDN: "ou=users,dc=example,dc=com"  # Base DN for LDAP users
+      mailAttributeName: "mail"  # Attribute for email in the LDAP schema
+      sslEnabled: true  # Enable SSL for secure LDAP
+      truststoreConfigType: "TrustAll"  # Trust store type (options: TrustAll, JVMDefault, HostName, CustomTrustStore)
+      trustStoreConfig:
+        trustAllConfig:
+          examineValidityDates: true  # Reject certificates outside the validity window
+  authorizer:
+    adminPrincipals: ["admin"]  # Add admin users here
+    principalDomain: "example.com"  # Organization domain for principal matching
+openmetadata:
+  server:
+    configOverrides:
+      security:
+        authProvider: ldap
+        jwtToken:
+          enabled: true  # Enable JWT tokens for secure communication
+          publicKeyUrls:
+            - "https://<your-domain>/api/v1/system/config/jwks"  # Replace with your domain
+
```

Comment:
Hi, This is missing the closing backticks (3 backticks) for Code Block. Please update this.